plugins/smart_proxy_remote_execution_ssh/remote_execution_ssh.rb

```diff
@@ -1 +1,2 @@
-gem 'smart_proxy_remote_execution_ssh', '0.1.3'
+gem 'smart_proxy_remote_execution_ssh', '0.1.4'
+gem 'foreman_remote_execution_core'
```

Comment:
I guess smart_proxy_remote_execution_ssh is loading foreman_remote_execution_core, so I hope it can get ommited here...